views/chat-view/src/components/MessageItem/MessageItem.tsx

```
@@ -101,7 +102,7 @@ export default function MessageItem({
                 size="300"
                 className={styles.body}
                 nomargin
-                dangerouslySetInnerHTML={{ __html: replyMessage?.body }}
+                dangerouslySetInnerHTML={{ __html: replyMessages[0]?.body }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)